### DISCUSSION TITLE
Please announce your favorite daily dish(side dish, 好きな惣菜)!

### DISCUSSION THREAD

**Discussion by mor-kana** [NAIST-SE/AttractingContributors#1051] [2024 Attraction Contributions]
Please tell your favorite daily dish (side dish)!
好きな総菜発表してください!

If you would like to help, check to [this link](https://github.com/mor-kana/AttractingContributors-DailyDish)!

**Comment by Ashmari** [comment]
Hi, I added my favorite dish .
I would be happy if you could also send a pull request to my repository!

https://github.com/Ashmari/AttractingContributors_BookVoyage.git
